docs/USAGE.md

```diff
@@ -298,3 +298,5 @@ To export an older version of a report, add the `report_date` parameter with a d
 *Quality-time* can send notifications about metrics that change status to Microsoft Teams channels. To enable notifications for a report, expand the report header and paste a [Microsoft Teams webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
 
 If a webhook has been configured, *Quality-time* will check for new red (target not met) and white (source unreachable or error parsing source data) metrics every minute. As soon as one or more metrics in the report change status, a notification will be sent to the Microsoft Teams channel configured by the webhook.
+
+For configurations that have a higher than default notification frequency any notifications generated before changes are send will still be send with the old configuration.
```

Comment:
```suggestion
To prevent too many notifications, it's possible to specify a notification frequency. Notifications will be bundled and sent once per period. Note that if you change the notification configuration, e.g. change a Microsoft Teams web hook, earlier notifications that haven't been sent yet, will still be sent using the old configuration.
```